frontend/src/hooks/useCheckout.js

```diff
@@ -0,0 +1,21 @@
+import { useContext } from 'react';
+import axios from 'axios';
```

Comment:
skip axios, I've implemented the basics with these so that you can skip config + auth + etc.

```suggestion
import { useAuth, useFetch, useServer } from '#hooks';
```